src/main/java/de/tum/in/www1/artemis/service/TutorParticipationService.java

```diff
@@ -171,15 +200,17 @@ private boolean isValidTutorialExampleSubmission(List<Feedback> tutorFeedback, L
     private void validateTutorialExampleSubmission(ExampleSubmission tutorExampleSubmission) {
         var tutorFeedback = tutorExampleSubmission.getSubmission().getLatestResult().getFeedbacks();
         var instructorFeedback = exampleSubmissionRepository.getFeedbackForExampleSubmission(tutorExampleSubmission.getId());
+        boolean equalFeedbackCount = instructorFeedback.size() == tutorFeedback.size();
 
-        if (isValidTutorialExampleSubmission(tutorFeedback, instructorFeedback)) {
-            return;
-        }
+        var unreferencedInstructorFeedbackCount = instructorFeedback.stream().filter(feedback -> feedback.getType() == FeedbackType.MANUAL_UNREFERENCED).toList().size();
+        var unreferencedTutorFeedback = tutorFeedback.stream().filter(feedback -> feedback.getType() == FeedbackType.MANUAL_UNREFERENCED).toList();
```

Comment:
Could we compare the length of both list directly here so we don't need to check it every time we execute the loop below?

Reply:
I was also thinking about this, but this complicated the validation logic, this is why I made it like this.